src/Loop/Driver.php

```diff
@@ -105,6 +105,8 @@ public function onSignal($signo, callable $callback, $data = null);
      * @param string $watcherId The watcher identifier.
      *
      * @return void
+     *
+     * @throws InvalidWatcherException Thrown if the watcher identifier is invalid or cancelled.
```

Comment:
I was just commenting that the PHPDoc is inconsistent with the other methods (which assume that invalid and cancelled are the same).